packages/generator/ni_measurement_plugin_sdk_generator/client/templates/measurement_plugin_client.py.mako

```diff
@@ -39,7 +39,7 @@ class Output(NamedTuple):
     """Measurement result container."""
 
     ${output_parameters_with_type}
-<% output_type = "Output" %>\
+<% output_type = "Optional[Output]" %>\
```

Comment:
This reduces usability because it requires callers to check whether the return value is None. Also, it highlights the fact that stream_measure doesn't allow the caller to determine whether the call was canceled or not.

Perhaps it would be better to propagate gRPC CANCELLED exceptions to the caller rather than trying to sweep them under the rug?

How will the LabVIEW client handle this? Does it indicate whether the call was canceled?

Reply:
Sure, I have removed the error suppression and updated to raise gRPC CANCELLED exception. 

In LabVIEW, the cancel is not implemented yet. It is still in prototype stage, where it is planned to raise error or show cancelation message alone.